capstone/src/main/java/com/google/sps/servlets/student/StudentServlet.java

```diff
@@ -56,13 +57,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Initally empty in case there is no club list
     ImmutableList<String> clubs = ImmutableList.of();
     if (currentStudent.getProperty(Constants.PROPERTY_CLUBS) != null) {
-      String clubsAsString = currentStudent.getProperty(Constants.PROPERTY_CLUBS).toString();
-
-      // Convert string representation of a list to an ImmutableList e.g. "[1, 2, 3]" -> [1, 2, 3]
-      // Substring to remove beginning and closing brackets of list
-      // Split by commas separating each element
       clubs =
-          ImmutableList.copyOf(clubsAsString.substring(1, clubsAsString.length() - 1).split(","));
+          ImmutableList.copyOf(
+              (ArrayList<String>) currentStudent.getProperty(Constants.PROPERTY_CLUBS));
```

Comment:
Does this need to be casted as ArrayList first? I'm not entirely sure and not a big deal either way, but I think .copyOf works on most iterables?

Reply:
I think it does because entity properties are not iterables, they're just Objects so the only other way I've seen to make it work is converting it to a string and then to ImmutableList as I did before.